packages/email/src/components/Block.tsx

```diff
@@ -24,7 +24,7 @@ const sizes = {
   medium: {
     min: 640,
     default: 641,
-    max: 999,
+    max: 991,
```

Comment:
Also, I made this change but was unaware of what the `email` package is. Perhaps this is an incorrect change.